WBooks/DetailBook/DetailBookViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  DetailBookViewController.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 25/08/21.
+//
+
+import UIKit
+
+class DetailBookViewController: UIViewController {
+    
+    private lazy var detailBookView = DetailBookView()
+    private let detailBookViewModel: DetailBookViewModel
+
+    init(detailBookViewModel: DetailBookViewModel) {
+        self.detailBookViewModel = detailBookViewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = NSLocalizedString("TITLE_DETAILBOOK", comment: "")
+        detailBookView.tableView.delegate = self
+        detailBookView.tableView.dataSource = self
+        detailBookView.bookImage.imageFromUrl(urlString: detailBookViewModel.book.image)
+        detailBookView.titleLabel.text = detailBookViewModel.book.title
+        detailBookView.authorLabel.text = detailBookViewModel.book.author
+        detailBookView.yearLabel.text = detailBookViewModel.book.year
+        detailBookView.genreLabel.text = detailBookViewModel.book.genre
+        detailBookView.statusLabel.text = detailBookViewModel.book.status
+        
+        detailBookView.rentButton.addTarget(self, action: #selector(rentPress), for: .touchUpInside)
+        detailBookView.addWishlistButton.addTarget(self, action: #selector(addWishlistPress), for: .touchUpInside)
+        
+        detailBookViewModel.getComments { comments in
+            if comments.count > 0 {
+                self.detailBookView.tableView.reloadData()
+            } else {
+                self.detailBookView.commentsView.isHidden = true
+            }
+        }
+    }
+    
+    override func loadView() {
+        view = detailBookView
+    }
+    
+    @objc func rentPress(sender: UIButton){
+        if (detailBookViewModel.validateStatusRent()) {
+            detailBookViewModel.rent(){ error in
+                if error == nil {
+                    self.showAlert(title: NSLocalizedString("TITLE_INFORMATION", comment: ""), message: NSLocalizedString("INFORMATION_SUCCESS", comment: ""), buttonDone: NSLocalizedString("BUTTON_DONE", comment: ""))
```

Comment:
Add space after each comma to improve the readability.
Like 
```
        self.showAlert(title: NSLocalizedString("TITLE_INFORMATION", comment: ""),
                       message: NSLocalizedString("INFORMATION_SUCCESS", comment: ""),
                       buttonDone: NSLocalizedString("BUTTON_DONE", comment: ""))
```

Reply:
Done